pages/shared/components-showcase/amp-autocomplete.html

```diff
@@ -0,0 +1,7 @@
+<amp-autocomplete filter="substring" id="myAutocomplete">
+  <p>Please enter "apple", "banana" or "cherry"</p>
+  <input />
+  <script type="application/json">
+    {"items": ["apple", "banana", "cherry"]}
+  </script>
+</amp-autocomplete>
```

Comment:
@sebastianbenz, wondering to what extent those should be localized. Can you advice?